lib/rubygems/gemcutter_utilities.rb

```diff
@@ -309,15 +310,34 @@ def pretty_host(host)
   end
 
   def get_scope_params(scope)
-    scope_params = {}
+    scope_params = { index_rubygems: true }
 
     if scope
       scope_params = { scope => true }
     else
-      say "Please select scopes you want to enable for the API key (y/n)"
-      API_SCOPES.each do |s|
-        selected = ask_yes_no(s.to_s, false)
-        scope_params[s] = true if selected
+      say "Here are the scope access we will use:"
+      scope_params.each do |k, _v|
+        say "  #{k}: y"
+      end
+      say "\n"
+      customise = ask_yes_no("Do you want to customise scopes?", false)
+      if customise
+        scope_params = {}
+        exclusive_params = {}
+        EXCLUSIVELY_API_SCOPES.each do |s|
+          selected = ask_yes_no(s.to_s, false)
+          exclusive_params[s] = true if selected
+          break if selected
+        end
+        if exclusive_params.any?
+          scope_params = exclusive_params
+          say "You selected one scope that must be enabled exclusively. Other scopes will be disabled."
```

Comment:
I think we should clarify this.

```suggestion
          say "You selected #{exclusive_scope}, which must be enabled exclusively. Other scopes will be disabled."
```

Another idea is to change this to a yes or no and ask if this is fine, if no then go onto the non exclusive scopes.

Reply:
Thank you, really appreciate it 😃 
Updated with the suggestions :D